api/tasks.py

```diff
@@ -4,25 +4,38 @@
 import wordpress
 from wordpress.user import WPUser
 from lazona_connector.celery import app
+from api.models import Shipment, DeliveryStatus
 
 
 @app.task
 def create_delivery(order):
     deliveries_by_vendor = Client(order).create_delivery()
     for delivery in deliveries_by_vendor:
+        label_url = ""
         if delivery._is_errored():
+            status_delivery = DeliveryStatus.ERROR_FROM_BODY
             FailedDeliveryMail(
                 order_id=delivery.order_id,
                 error_returned=delivery.data.get("mensaje"),
                 req_body=delivery.req_body
             ).send()
         else:
+            status_delivery = DeliveryStatus.LABEL_SENT
+            label_url = delivery.print_pdf()
             SuccessDeliveryMail(
-                pdf_path=delivery.print_pdf(),
+                pdf_path=label_url,
                 recipient=delivery.vendor.email,
                 order_id=delivery.order_id
             ).send()
 
+        Shipment(
+            shipment=delivery.data.get("codBarras"),
+            order=int(delivery.order_id),
+            vendor=int(delivery.vendor.id),
```

Comment:
sorry for being so picky, but if they are ids, I'd name the columns like `order_id` and `vendor_id`. Future devs will thank us for it.

Reply:
No worries, u are right.